extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1087,7 +1093,7 @@ export class AmpA4A extends AMP.BaseElement {
     dev().assert(!!this.element.ownerDocument, 'missing owner document?!');
     this.protectedEmitLifecycleEvent_('renderFriendlyStart');
     // Create and setup friendly iframe.
-    const iframe = /** @type {!HTMLIFrameElement} */(
+    this.iframe = /** @type {!HTMLIFrameElement} */(
```

Comment:
Make sure to reset to null with unlayoutCallback after children elements are removed.  Actually, we shouldn't necessarily be removing all children as that would remove the placeholder and responsive layout filler element.  In reality, we want to remove only the iframe and any potentially created amp analytics element (which should be removed by adsense/doubleclick impls overriding unlayout callback and doing so after calling super).